spec/classes/check_mk_agent_spec.rb

```diff
@@ -1,8 +1,40 @@
 require 'spec_helper'
 
 describe 'check_mk::agent' do
-  on_supported_os.each do |os, os_facts|
-    let(:facts) { os_facts }
+  on_supported_os.each do |os, facts|
+    systemd_facts = case facts[:operatingsystem]
```

Comment:
Given the repetition, perhaps this should live in a spec helper? The `systemd_version` fact can be a nice addition here:
https://github.com/voxpupuli/voxpupuli-test/blob/f6c85fe54412248339cbb8ebe197c0b95dbb5697/lib/voxpupuli/test/facts.rb#L53-L54

https://github.com/voxpupuli/modulesync_config/pull/693 will then be needed to use it, but it can save a lot of duplication.